src/spokanetech/settings.py

```diff
@@ -39,6 +39,7 @@
 
     DEBUG = False
     ALLOWED_HOSTS = ["spokanetech.org", "spokanetech-py.fly.dev"]
+    CSRF_TRUSTED_ORIGINS = ["https://" + host for host in ALLOWED_HOSTS]
```

Comment:
Any reason for string concatenation over f-strings? Not a nit, just curious.

Reply:
I typically prefer f-strings, so not sure why I used concatenation here. I'll update to use f-strings